Mathlib/Data/Polynomial/HasseDeriv.lean

```diff
@@ -228,12 +228,12 @@ section
 open AddMonoidHom Finset.Nat
 
 theorem hasseDeriv_mul (f g : R[X]) :
-    hasseDeriv k (f * g) = ∑ ij in antidiagonal k, hasseDeriv ij.1 f * hasseDeriv ij.2 g := by
+    hasseDeriv k (f * g) = ∑ ij in Finset.antidiagonal k, hasseDeriv ij.1 f * hasseDeriv ij.2 g := by
```

Comment:
`open Finset (antidiagonal)` on the line above would save the need to add namespacing here.